Tella/Domain/Entity/Report/Project.swift

```diff
@@ -18,6 +18,7 @@ class Server : Hashable {
     var slug : String?
     var autoUpload: Bool?
     var autoDelete: Bool?
+    var serverType: Int?
```

Comment:
Could you use the `ServerType` enum  instead of `Int`

Reply:
@dhekra-rouatbi  Could you remove the ServerType and only work with ServerConnectionType and add unknown case. you mean ServerConnectionType here ?